tech-committee-charter.md

```diff
@@ -65,6 +68,17 @@ In such cases, TC members will inform the the wider TC of the perceived conflict
 
 ### Election of TC Members
 
+New TC members can be nominated by any Member in Standing as defined in the [OpenFeature Governance Charter](https://github.com/open-feature/community/blob/main/governance-charter.md), and must be sponsored by one of the existing TC members to bring it to a vote. A candidate can be elected to the TC by the super-majority vote (greater than two thirds) of the existing TC members. The voting by the TC members is anonymous. A multiple-candidate method, such as [Condorcet](http://en.wikipedia.org/wiki/Condorcet_method) or [Single Transferable Vote](http://en.wikipedia.org/wiki/Single_transferable_vote), may be used to conduct the vote, by phrasing the selections as:
```

Comment:
Do we really need a complex voting method. Majority from the GC should be fine.

Reply:
Can we please simplify the voting as discussed on the last GC call:

New TC members must be nominated by existing TC members. GC has a veto for new members. 

The TC servers as the technical arm of the GC. The GC is already voted by the wider community, so no need to make everything a community vote here.